src/monodroid/jni/embedded-assemblies.cc

```diff
@@ -208,6 +209,10 @@ EmbeddedAssemblies::typemap_java_to_managed (const char *java_type_name)
 	}
 
 	const char *managed_type_name = entry->to;
+	if (managed_type_name == nullptr) {
+		log_debug (LOG_DEFAULT, "typemap: Java type '%s' maps either to an open generic type or an interface type.");
```

Comment:
This should probably use `LOG_ASSEMBLY` as `LOG_DEFAULT` requires an `adb shell setprop debug.mono.log default` to see the messages, which just *looks* weird.